### ISSUE TITLE
Bug: Incorrect color pattern for react badge on React Lesson

### ISSUE THREAD

**Issue by 0xabdulkhalid** [TheOdinProject/theodinproject#4088]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `Bug: brief description of bug` format, e.g. `Bug: Lesson complete button does not update on click`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Bug:**
- The Bug is about the incorrect color pattern used for React Badge which can be found on Javascript Learning Path page.
- Our Odin Project primarily used this color `#E29E7A` for every badges. 

**2. How To Reproduce:**
1. Sign in
2. Visit a React lesson on Full-Stack JavaScript page
3. Observe the incorrect color pattern for React Badge by comparing it with other badges.

**3. Expected Behavior:**
- The color of the React badge need to be the same as other badges.

**4. Desktop/Device:**
-   Device: Lg Q Stylus
-   OS: Android 8 Oreo
-   Browser: Opera Mobile
-   Version: 76.1

**5. Additional Information:**
- You can simply compare this one's color pattern with other badges to see the difference.
- I would like to share screenshots taken from Odin Site, you can able to see the changes with React badge and Node badge.

|REACT BADGE|NODE BADGE|
|:---:|:---:|
|![image](https://github.com/TheOdinProject/theodinproject/assets/92252895/e1931e25-2f35-4be2-8022-d4b7364434e4)|![image](https://github.com/TheOdinProject/theodinproject/assets/92252895/d867f909-a19c-46f7-8ee2-063a7461482f) |

**Comment by KevinMulhern** [comment]
Thanks for doing reporting 💪  

Matching the other badge colors within the React SVG would be the ideal solution. I had a bit of a play and I think it should be easy.

**The "React" text in the badge:**
For all the path elements from line 3 [here](https://github.com/TheOdinProject/theodinproject/blob/main/app/assets/images/badge-react.svg?short_path=8801c37#L3) to line 7. We just need to change the fill values from `#CE973D` to `#D59E6F`

**The React logo:**
For all the path elements from line 9 [here](https://github.com/TheOdinProject/theodinproject/blob/main/app/assets/images/badge-react.svg?short_path=8801c37#L9) to line 12. We also just need to change the fill values from `#CE973D` to `#D59E6F`.

We have a [borderless version of this badge](https://github.com/TheOdinProject/theodinproject/blob/main/app/assets/images/badges/badge-react-borderless.svg) (used in the progress circles) that will also need to get the exact same updates as above.

[A PR](https://github.com/TheOdinProject/theodinproject/pull/4089) was opened before any  Odin maintainers had a chance to triage this. But the fix isn't quite right so we'll close it. Would you still like to be assigned to this @0xabdulkhalid?

**Comment by KevinMulhern** [comment]
All yours @0xabdulkhalid

**Comment by KevinMulhern** [comment]
Hey @0xabdulkhalid, any update on this? do you still want to work on it or would you like us to open it up for someone else to take on?

**Comment by 0xabdulkhalid** [comment]
Yep @KevinMulhern, I'm still want to work on this issue.

Currently I'm on my exams, They are going about to end. So i will complete this issue within 1 week.

**Comment by KevinMulhern** [comment]
Hey @0xabdulkhalid any updates? would you like me to assign someone else if you don't have the time?

**Comment by 0xabdulkhalid** [comment]
Yeah @KevinMulhern, You're right. I have no time to work on this issue due to my exams.

You can assign this issue to someone in order to fix this issue.

Sorry for the inconvenience, Thank you for your support.

**Comment by KevinMulhern** [comment]
No worries @0xabdulkhalid, thanks for letting us know and best of luck with your exams mate 💪

**Comment by Asartea** [comment]
Could I be assigned this?

**Comment by ManonLef** [comment]
@KevinMulhern as mentioned in our discord discussion, I've edited these in Adobe Illustrator:

 I've expanded the lines for a quick fix. Not sure if that would be an issue in the future (in case we need any fine control over the line widths and such. 

I also took the liberty to fix something that bugged me for a while. Everything inside the circle looked uncentered to me, because the inner not-full circle was up from center compared to outer. 

I'd not mind considering a scalable option for all of these as TOP assets. That would mean redoing all of them in the same easily editable fashion. Not something I'd have time for now, but it would make the icons overall a lot more consistent. The NodeJS one for example has rounded line caps, and line weights are not consistent across the different badge designs. 
So for the future: I'd be happy to take on the rework/re-design of all badge assets if time permits.

## Reworked and Centered more: 
![badge-react-centered-01](https://github.com/TheOdinProject/theodinproject/assets/81025586/e49b39fb-6bfa-483b-9b66-a80f8bccff39)

## Side by side, old on the left, centered right:
![Screenshot 2023-09-17 at 17 53 13](https://github.com/TheOdinProject/theodinproject/assets/81025586/ac6ff853-c7c1-4ee2-bb1a-63a5debec15b)

**Comment by ManonLef** [comment]
I will also work on the one without the outer ring

**Comment by ManonLef** [comment]
@KevinMulhern an updated version to match the gradient breakpoints with the NodeJS badge

![badge-react-centered-v2](https://github.com/TheOdinProject/theodinproject/assets/81025586/bb0fdc47-8cb9-41fa-a8bb-cd3f11d1ab7e)

**Comment by Asartea** [comment]
oooooh, pretty

Really nice work Manon

**Comment by ManonLef** [comment]
Thanks Asartea! Still working on tweaking sadly :D They don't quite behave as expected (yet)!

**Comment by ManonLef** [comment]
Fingers crossed @KevinMulhern 🫣

![badge-react-V3-01](https://github.com/TheOdinProject/theodinproject/assets/81025586/3d5f3606-1e4a-43f5-b77d-c2b812994cad)

**Comment by KevinMulhern** [comment]
Looks really good to me @ManonLef, nice work 💪 

<img width="915" alt="Screenshot 2023-09-17 at 22 00 15" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/63b51813-e477-4ab4-8abd-deb79c43591b">

The middle looks a tiny bit redder than the other badge. But I think thats just a consequence of the gradient and how much is happening in the middle of this particular badge. Plus, I've probably been staring this too long tonight and it's without a doubt 100x better than what we have on the site right now. I'm more than happy to roll this out if you are?

**Comment by ManonLef** [comment]
@KevinMulhern I am as well. I noticed the more visible pink. It's indeed due to the amount of surface showing the gradient in the center there. I tested with changing the gradient breakpoint percentages but that would make it overly yellow in my opinion from the top left down again. I think we should roll this out indeed. If we get to redoing them all we can make them more coherent :)

**Comment by KevinMulhern** [comment]
> If we get to redoing them all we can make them more cohesive :)
Sounds good to me 💯 

We need a borderless version of the badge before opening this up for someone to work on. I can do that myself to save you the trouble. Thanks again for all the work you've put into this Manon.

**Comment by ManonLef** [comment]
Oh yes I forgot about the borderless one @KevinMulhern 
I'm on it!

**Comment by ManonLef** [comment]
![badge-react-V3-01-borderless](https://github.com/TheOdinProject/theodinproject/assets/81025586/2c3260f6-aaf6-4578-8c3e-205dadbf8613)

**Comment by KevinMulhern** [comment]
That was rapid 🚤  you're the best Manon!

@Asartea would you still be interested in taking this one? 👀

**Comment by Asartea** [comment]
@KevinMulhern yes; apologies for the delay, was very busy recently
